packages/react-cards/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
```

Comment:
Not sure this package needs its own launch.json?

Reply:
Yeah, it's probably fine if we remove it. I'll take it out and, in case it is needed, will come back and comment that.